src/graphql/data/RelayEnvironment.ts

```diff
@@ -1,16 +1,38 @@
 import ms from 'ms.macro'
-import { Environment, Network, RecordSource, Store } from 'relay-runtime'
-
-import fetchGraphQL from './fetchGraphQL'
+import { RelayNetworkLayer, retryMiddleware, urlMiddleware } from 'react-relay-network-modern'
+import { Environment, RecordSource, Store } from 'relay-runtime'
 
 // This makes it possible (and more likely) to be able to reuse data when navigating back to a page,
 // tab or piece of content that has been visited before. These settings together configure the cache
 // to serve the last 250 records, so long as they are less than 5 minutes old:
 const gcReleaseBufferSize = 250
 const queryCacheExpirationTime = ms`5m`
 
+const GRAPHQL_URL = process.env.REACT_APP_AWS_API_ENDPOINT
+if (!GRAPHQL_URL) {
+  throw new Error('AWS URL MISSING FROM ENVIRONMENT')
+}
+
+const MAX_RETRIES = 10
+const network = new RelayNetworkLayer([
```

Comment:
Can you add a note that the network layer must not cache, or it will break cache-evicting network policies (like network-only)?

Also check that this new class does not cache by default.